x-pack/plugins/security_solution/public/common/components/event_details/insights/insights.tsx

```diff
@@ -45,14 +49,14 @@ export const Insights = React.memo<Props>(
       { category: 'kibana', field: 'kibana.alert.rule.parameters.index' },
       data
     );
-    const hasProcessEntityInfo =
-      isRelatedAlertsByProcessAncestryEnabled && processEntityField && processEntityField.values;
+    const hasProcessEntityInfo = processEntityField && processEntityField.values;
```

Comment:
Would you be willing to confirm that we still want to remove the `isRelatedAlertsByProcessAncestryEnabled` check?

Reply:
nvm, it just migrated south for the winter to this check:

```ts
const hasProcessSessionInfo =
      isRelatedAlertsByProcessAncestryEnabled && processSessionField && processSessionField.values;
```